### DISCUSSION TITLE
弹出“请登录提示框”

### DISCUSSION THREAD

**Discussion by mklBingo** [umijs/umi#10862] [Q&A]
如何在公共页面进行菜单切换时，如检测到未登录状态，先弹出modal框，询问是否确认登录，选择后进行路由的跳转或者取消登录？

**Comment by fz6m** [comment]
自己封装 axios 响应拦截器，发现 code 是没登录权限就跳转到 sso 去，带着回跳 url 参数，等登录完了 sso 会自动跳回来。

如果没有 sso ，可以换成登录页，一个道理。
